bin/build-drivers/deps.edn

```diff
@@ -11,7 +11,6 @@
   io.forward/yaml                 {:mvn/version "1.0.9"} ; Don't upgrade yet, new version doesn't support Java 8 (see https://github.com/owainlewis/yaml/issues/37)
   io.github.clojure/tools.build   {:git/tag "v0.7.4" :git/sha "ac442da"}
   org.clojure/tools.deps.alpha    {:mvn/version "0.12.985"}
-  org.flatland/ordered            {:mvn/version "1.5.9"} ; used by io.forward/yaml -- need the newer version
```

Comment:
@camsaul I wanted to check with you on this one. The version pulled in by this `deps.edn` causes a failure in `bin/build-drivers` tests. Removing this solves it, but so would just adding the same version that is in the root `deps.edn`. I'm not clear on why there is an explicit dependency on this lib in this project. Without removing or updating flatland here, the test throws an error like:

```
Step failed: Syntax error compiling at (metabase/query_processor/middleware/cache/impl.clj:44:20).
{:via
 [{:type clojure.lang.Compiler$CompilerException,
   :message "Syntax error compiling at (metabase/query_processor/middleware/cache/impl.clj:44:20).",
   :data
   #:clojure.error{:phase :compile-syntax-check,
                   :line 44,
                   :column 20,
                   :source "metabase/query_processor/middleware/cache/impl.clj"},
   :at [clojure.lang.Compiler analyze "Compiler.java" 6825]}
  {:type java.lang.RuntimeException,
   :message "No such var: ordered-map/ordered-map-reader",
   :at [clojure.lang.Util runtimeException "Util.java" 221]}],
 :trace
 ...
 ```

Reply:
> I'm not clear on why there is an explicit dependency on this lib in this project.

```clj
; used by io.forward/yaml -- need the newer version
```

that's why

Reply:
bumping it to the latest version is good, if it works now if you remove it then that's fine too